src/wiki/keyboard-shortcuts.md

```diff
@@ -0,0 +1,208 @@
+﻿# Keyboard shortcuts
+
+Mastering keyboard shortcuts can significantly enhance your productivity on Windows.
+
+This article will introduce you to the essential shortcuts that will help you navigate the new operating system with ease.
+
+# General shortcuts (Applies to all versions of Windows)
```

Comment:
```suggestion
## General shortcuts (Applies to all versions of Windows)
```

For better accessibility, only use H1s for the article title, and to make the "On this page" functionality working properly.